cmd/virt-handler/virt-handler.go

```diff
@@ -216,6 +215,9 @@ func (app *virtHandlerApp) AddFlags() {
 	flag.StringVar(&app.VirtShareDir, "kubevirt-share-dir", virtShareDir,
 		"Shared directory between virt-handler and virt-launcher")
 
+	flag.StringVar(&app.CertDir, "cert-dir", certificateDir,
+		"Shared directory between virt-handler and virt-launcher")
```

Comment:
? what is this for?

Reply:
The description should be something like `Certificate store directory`. Has nothing to do with virt-launcher. Copy-paste error.